src/pudl/transform/ferc1.py

```diff
@@ -96,25 +97,31 @@ class Ferc1RenameColumns(TransformParams):
       e.g. in the case of unit conversions with a column rename.
     """
 
-    dbf: RenameColumns = {}
-    xbrl: RenameColumns = {}
+    dbf: RenameColumns = None
+    xbrl: RenameColumns = None
+    duration_xbrl: RenameColumns = None
+    instant_xbrl: RenameColumns = None
```

Comment:
Why is it appropriate / preferable to give it `None` as an argument here instead of using the default constructor as we seem to be doing in many other cases, i.e.

```py
    dbf: RenameColumns = RenameColumns()
    xbrl: RenameColumns = RenameColumns()
    duration_xbrl: RenameColumns = RenameColumns()
    instant_xbrl: RenameColumns = RenameColumns()
```

As it is there's a type mismatch, since `RenameColumns` can't be instantiated with `None` as its argument -- it requires `dict[str,str]` (which the empty dictionary satisfies). I'm surprised Pydantic isn't complaining about this, and I wonder if it's just that there's nowhere in the code right now that's attempting to instantiate a `RenameColumnsFerc1` object using the default parameters.

What happens if you try and create a `RenameColumnsFerc1` object with no arguments in a notebook, with `None` as the default values above?

Reply:
I am also a little suprised about this. I was getting an error when I had these empty `RenameColumns`'s as the default when they weren't being overridden with the params so I tried `None` and it worked fine. But I 100% agree that these should be empty `RenameColumns()` and I have reverted this change and tested things and they seem to work fine.

Not sure why `None` wasn't raising issues....